### ISSUE TITLE
Virtual Copy Paste

### ISSUE THREAD

**Issue by akash435** [prathimacode-hub/Awesome_Python_Scripts#581]
---
Title: Virtual Copy Paste
About: It's a GUIscript 
Name: Akash Ramanand Rajak
Label: feature request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

In this script,
will create two textarea and two buttons, one for COPY and other for PASTE.

these two buttons will be implemented in such a way we can write any thing in one textarea and copy that using COPY button,
and using paste button we can paste the copied things in other text area.

So basically, here copy and paste will be done using COPY and PASTE button instead if CTRL+C and CTRL+V.

**Comment by akash435** [comment]
@prathimacode-hub , please assign this to me under LGM SOC

**Comment by Debashish-hub** [comment]
Issue is assigned to you @akash435
